### PR TITLE
[WIP] Add starting README

### PR DESCRIPTION
**Pull request by pinin4fjords**
Working on our walkthrough

Edit: now incorporating content from https://github.com/robsyme/go-with-the-flow/pull/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+Change to the 'cli' directory by typing the following into the terminal:
+
+```
+cd activity/cli
+```
+
+Now, try to assign an image to a class, for example: 
+
+```
+classify --image data/rain-ready.png --labels 'cat,dog,cute dog'
```
Comment (maxulysse) [inline README.md:61]: I'm getting:
> `bash: classify: command not found`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- Reproducibility
+- Resumability
+- Reporting
+- Flexibility
+
+This knowledge will enhance your ability to connect with our customers, addressing their needs and inquiries with greater empathy and insight.
+
+## Environment setup
+
+For this workshop we're going to be using a tool called 'GitPod', which provides us with a ready-made software environment to play in. You will need a GitHub account, so if you don't have one, go to [GitHub](https://github.com/) and create on for yourself.
```
Comment (laramiellindsey) [inline README.md:20]:
```suggestion
For this workshop we're going to be using a tool called 'GitPod', which provides us with a ready-made software environment to play in. You will need a GitHub account, so if you don't have one, go to [GitHub](https://github.com/) and create one for yourself.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+It will make things easier for you if you have a browser extension to provide easier access to GitPod. Go [here](https://www.gitpod.io/docs/configure/user-settings/browser-extension) and follow the link to install the extension for your preferred browser. Restart your browser if required.
+
+### Start the GitPod environment
+
+Now, from this repository, you should see a green button with a funny-shaped 'G', to the right of the 'code' button. Click that, follow any prompts, wait, and you should get a GitPod workspace with all the right things installed.
+
+If you're not used to a command line environment, don't worry, you'll just be copying commands to follow along. But you can do things like:
+
+```bash
+cd data # Change directory
```
Comment (laramiellindsey) [inline README.md:33]:
```suggestion
cd data # Change directory to data
```

File: docs/cli.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
+# Critter classification with bare command line working 
+
+In this section of our Nextflow tutorial, we'll explore how to run an existing classification model using OpenAI's CLIP directly from the command line, to both classify our critters and build a collage of images.
+
+We're assuming that you are successfully in an environment where the files in this repository have been installed and the necessary software made available. Please go back and look at the environment setup in the README if that's not the case.
```
Comment (laramiellindsey) [inline docs/cli.md:5]:
```suggestion
We're assuming that you are successfully in an environment where the files in this repository have been installed and the necessary software made available. Please go back and look at the environment setup in the [README](../README.md) if that's not the case. 
```

File: docs/cli.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+montage -geometry 80 -tile 4x dogs/* tmp.png
+montage -label '$label' tmp.png -geometry +0+0 -background Gold collages/dog.png
+rm tmp.png
+```
+
+## Step 3 combine the collages
+
+You should now have a set of collages, one for each critter type, in the `collages` directory.
+
+All you need to do now is combine them, which which you can do with the following command:
```
Comment (laramiellindsey) [inline docs/cli.md:68]:
```suggestion
All you need to do now is combine them, which you can do with the following command:
```

File: docs/bash.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+ * This is clearly a little more reproducible because we're removing some human involvement. But what if someone in the host environment changes what the 'classify' command does?
+ * If this was a more time-consuming task where each image took 10 minutes to classify and there is a power failure, would we be able to resume the sequence of commands. 
+ * How well will this scale to 1000s of images?
+ * What would happen if you copied this script to a different computer where the software hasn't already been installed?
+
+<details>
+<summary>Summary</summary>
+While Bash scripts offer more efficiency and scalability over running individual CLI commands, their is still an important limitation with respect to reproducibility and scalability. For instance, executing the same script across different computers, or environments might yield varying results due to differences in software versions, operating systems, or configurations. Moreover, Bash scripts can become unwieldy as the complexity of the tasks increases, especially when dealing with large datasets or requiring parallel processing.
```
Comment (laramiellindsey) [inline docs/bash.md:38]:
```suggestion
While Bash scripts offer more efficiency and scalability over running individual CLI commands, there is still an important limitation with respect to reproducibility and scalability. For instance, executing the same script across different computers, or environments might yield varying results due to differences in software versions, operating systems, or configurations. Moreover, Bash scripts can become unwieldy as the complexity of the tasks increases, especially when dealing with large datasets or requiring parallel processing.
```

**Review by laramiellindsey — COMMENTED** [review]
Minor edits. Looking good so far!

**Comment by pinin4fjords** [comment]
Closing in favour of https://github.com/robsyme/go-with-the-flow/pull/4

**Pull request closed without merging**